tests/plugins/Qukeys/basic/sketch.h

```diff
@@ -0,0 +1,7 @@
+// -*- mode: c++ -*-
+
```

Comment:
There's no copyright header in the `sketch.*` files; @obra, do you care about that?